test/e2e/specs/linksConsolErros.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { ConsoleMessage, Page, expect } from '@playwright/test';
+import { vpTest } from '../fixtures/vpTest';
+import { LINKS } from '../testData/pageLinksData';
+
+/**
+ * Console error test generated by LINKS object array data.
+ */
+for (const link of LINKS) {
+    vpTest(`Test console errors on link ${link.name}`, async ({ page, consoleErrors, vpExamples }) => {
+        await vpExamples.clickLinkByName(link.name);
+        await waitForPageToLoadWithTimeout(page, 5000);
+        expect(page.url()).toContain(link.endpoint);
+        handleCommonBrowsersErrors(link.name, consoleErrors);
+    });
+}
+
+/**
+ * Testing number of links in page.
+ */
+vpTest('Link count test', async ({ page }) => {
+    const expectedNumberOfLinks = 37;
+    const numberOfLinks = await page.getByRole('link').count();
+    expect(numberOfLinks).toBe(expectedNumberOfLinks);
+});
+
+/**
+ * Helper function to handle common browser errors.
+ */
+function handleCommonBrowsersErrors(linkName: string, consoleErrors: ConsoleMessage[]) {
+    switch (linkName) {
+        case 'Custom Errors':
+            validatePageErrors(
+                consoleErrors,
+                ['(CODE:999 undefined) My custom error message'],
+                ['No compatible source was found for this media', 'Video cannot be played Public ID snow_horses not found', 'the server responded with a status of 404', 'Cannot read properties of undefined']
+            );
+            break;
+        case 'Debug mode':
+            validatePageErrors(consoleErrors, ['invalid player configuration', `cloudinary video player: \'fluid\' should be a boolean`], []);
+            break;
+        case 'VAST & VPAID Support':
+            validatePageErrors(consoleErrors, ['The Cross-Origin-Opener-Policy header'], ["Blocked script execution in 'about:blank' because the document's frame is sandboxed and the 'allow-scripts' permission is not set"]);
```

Comment:
I don't see this CORS error, you might choose to ignore it, but it is not expected, i.e. not part of the test

Reply:
nvm, I see it in localhost

Reply:
When running locally the error appears so we need to handle it.